_sass/_jumbotron.scss

```diff
@@ -38,6 +43,18 @@
   line-height: 4rem;
 }
 
+.jumbotron .tagline-0 {
```

Comment:
@AndreiZiatiev please fix the code formatting of this block and add the reason message here.

Reply:
this code is unneeded here. Removed